src/mon/MDSMonitor.cc

```diff
@@ -1647,6 +1647,13 @@ int MDSMonitor::filesystem_command(
     }
   } else if (prefix == "mds rmfailed") {
     mds_rank_t who;
+    string confirm;
+    if (!cmd_getval(g_ceph_context, cmdmap, "confirm", confirm) ||
+	confirm != "--yes-i-really-mean-it") {
+	  ss << "WARNING: this will cause failover mechanism to stop working. "
```

Comment:
We should make this message more severe.

Maybe "WARNING: this can make your filesystem inaccessible, use extreme caution!"